### ISSUE TITLE
Urllib import error in new release

### ISSUE THREAD

**Issue by SabobaS** [Yelp/elastalert#2385]
Whenever I try to run the new Elastalert release (0.2.0) I got an error coming from the import statements in "kibana.py" (urllib.error, urllib.parse urllib.request).

From the official documentation in [https://docs.python.org/2/library/urllib.html
]:
"Note: The urllib module has been split into parts and renamed in Python 3 to urllib.request, urllib.parse, and urllib.error. The 2to3 tool will automatically adapt imports when converting your sources to Python 3."

So, I guess that in order to NOT break backward compatibility with Python2, we should keep the old import statement.

**Comment by SabobaS** [comment]
An update. Raising an exception with:
`raise EAException("Error connecting to JIRA: %s" % (str(e)[:1024])).with_traceback(sys.exc_info()[2])`
(in alerts.py)
we are exploiting Python3 syntax. Therefore, I am getting an error in Python2.

For example, I noticed that this problem is in JiraAlerter but not in Slack one.

**Comment by yichingwu** [comment]
I have the same issue.

**Comment by Qmando** [comment]
I'm not going to be maintaining python 2 support going forward, so you'll need to use version < 2.

**Comment by yichingwu** [comment]
@Qmando , got it.
thanks.

**Comment by SabobaS** [comment]
Ok, but such a big change should be clearly written in the release notes.
Thank you for your answer.

**Comment by Qmando** [comment]
You're right. Haven't even updated the release notes at all yet, but I'm doing that now. Sorry about the lack of announcement.

**Comment by efinal** [comment]
should read this issue earlier...
going to reinstall docker with python 3 ...

**Comment by vsabelli** [comment]
@efinal 
For you and anyone who might be concerned: since we use Elastalert in my company, I am currently supporting a fork of this project. 
The goal is to maintain bugfixes and enhancements, keeping backward compatibility with Python2.

This is the project here in Github: https://github.com/vsabelli/elastalert

@Qmando, I am going to update the project description as soon as possible to specify the reason behind that project and give official Elastalert the credits it deserves.

**Comment by efinal** [comment]
@vsabelli great for your fork with python2 support, and I should read your response earlier... , I reinstall docker with python3 and make several changes to make elastalert 0.2.1 as well as the kibana plugin running with python3.
